src/gc.cr

```diff
@@ -1,13 +1,76 @@
+# :nodoc:
+fun __crystal_malloc(size : UInt32) : Void*
+  GC.malloc(LibC::SizeT.new(size))
+end
+
+# :nodoc:
+fun __crystal_malloc_atomic(size : UInt32) : Void*
+  GC.malloc(LibC::SizeT.new(size))
```

Comment:
What's the point if `__crystal_malloc_atomic` & `__crystal_malloc` have the same implementation?
It doesn't look right.